src/commands/registerCommands.ts

```diff
@@ -31,7 +31,7 @@ export function registerCommands(): void {
     // container apps
     registerCommandWithTreeNodeUnwrapping('containerApps.createContainerApp', createContainerApp);
     registerCommandWithTreeNodeUnwrapping('containerApps.deleteContainerApp', deleteContainerApp);
-    registerCommandWithTreeNodeUnwrapping('containerApps.deployImage', deployImage);
+    registerCommandWithTreeNodeUnwrapping('containerApps.deployImage', deployFromRegistry);
```

Comment:
Deferring doing any command renaming/remapping for now, we can handle that in Megan's upcoming deploy code